hubblestack_nova_profiles/cis/centos-7-level-1-scored-v2-1-0.yaml

```diff
@@ -1,164 +1,21 @@
-# NOTE: This CIS Profile only includes Level 1 Scored Items for CentOS 7
-# NOTE: Within this file, there are a few sections that should be tailored to your
-#       organization's specific policy.  Search for '# NOTE: ' comments through the file.
-
-# TODO: Checks that aren't implemented yet:
```

Comment:
So this means we have implemented all these checks, if not please have this section for the remaining checks.